docs/unified-test-documentation/test-matrix.md

```diff
@@ -0,0 +1,148 @@
+| 9.   | USB HID and MSC Support                           | All                                  |
+| 10.  | Debian Stable and Ubuntu LTS support              | All                                  |
+| 11.  | UEFI Shell                                        | All                                  |
+| 12.  | Windows 11 booting                                | All                                  |
+| 13.  | Audio subsystem                                   | All                                  |
+| 14.  | USB-C/Thunderbolt                                 | All                                  |
+| 15.  | Network boot                                      | All                                  |
+| 16.  | M.2 WiFi/Bluetooth                                | All                                  |
+| 17.  | SD card support                                   | All                                  |
+| 18.  | USB Camera verification                           | All                                  |
+| 19.  | Custom fan curve                                  | FAN001.001                           |
+| 20.  | SMBIOS                                            | DMI002.001, DMI003.001, DMI005.001,  |
+|      |                                                   | DMI006.001, DMI007.001, DMI008.001   |
+
+### Module: dasharo-security
+
+| No.  | Supported test suite                              | Supported test cases                 |
+|:----:|:-------------------------------------------------:|:------------------------------------:|
+| 1.   | TPM Support                                       | All                                  |
+| 2.   | Verified Boot support                             | VBO001.002, VBO002.002, VBO003.001   |
+| 3.   | Measured Boot support                             | All                                  |
+| 4.   | Secure Boot support                               | All                                  |
+
+## Test matrix - Talos 2
+
+### Module: dasharo-compatibility
+
+| No.  | Supported test suite                  | Supported test cases                           |
+|:----:|:-------------------------------------:|:----------------------------------------------:|
+| 1.   | coreboot base port                    | CBP001.001, CBP002.001, CBP004.001             |
+
+## Test matrix - Tuxedo IBS15
+
+### Module: dasharo-compatibility
+
+| No.  | Supported test suite                              | Supported test cases                 |
+|:----:|:-------------------------------------------------:|:------------------------------------:|
+| 1.   | Memory HCL                                        | All                                  |
+| 2.   | UEFI compatible interface                         | All                                  |
+| 3.   | Display ports and LCD support                     | All                                  |
+| 4.   | Embedded Controller and Super I/O initialization  | Whithout ECR010.001 and ECR010.002   |
```

Comment:
IMO we should decide either we use allow list or deny list, not both. So wither we write all tests which are supported from given suite, or we write all which are not supported. using `Without` breaks the pattern.